enterprise/core-edge/src/main/java/org/neo4j/coreedge/raft/roles/Candidate.java

```diff
@@ -101,7 +101,7 @@ else if ( res.term() < ctx.term() || !res.voteGranted() )
                             ctx.term(), ctx.myself(), outcome.getVotesForMe() );
 
                     outcome.setLeader( ctx.myself() );
-                    Leader.appendNewEntry( ctx, outcome, new NewLeaderBarrier() );
+                    Appending.appendNewEntry( ctx, outcome, new NewLeaderBarrier() );
```

Comment:
I think the previous solution was clearer actually, showing that appending a new entry is a Leader only operation, and keeping the code within the Leader class.